src/__tests__/Overlay.tsx

```diff
@@ -27,10 +27,8 @@ const TestComponent = ({initialFocus, callback}: TestComponentSettings) => {
         open overlay
       </Button>
       <Button>outside</Button>
-      {isOpen && (
+      {isOpen ? (
```

Comment:
why this change?

Reply:
I changed all the && to ternary based on https://github.com/primer/components/pull/1135#discussion_r614371832.  Could definitely have done it in a separate PR, but there were only a few